homeassistant/components/light/yeelight.py

```diff
@@ -85,7 +84,7 @@ def setup_platform(hass, config, add_devices, discovery_info=None):
             device = {'name': device_config[CONF_NAME], 'ipaddr': ipaddr}
             lights.append(YeelightLight(device, device_config))
 
-    add_devices(lights, True)  # true to request an update before adding.
+    add_devices(lights, True)
```

Comment:
That's covered in the [docs](https://home-assistant.io/developers/code_review_platform/). No need to have that on a single place in the code.